testsuite/integration-arquillian/tests/base/src/test/java/org/keycloak/testsuite/client/ClientPolicyBasicsTest.java

```diff
@@ -515,11 +515,11 @@ public void testMultiplePolicies() throws ClientRegistrationException, ClientPol
         createPolicy(policyBetaName, DefaultClientPolicyProviderFactory.PROVIDER_ID, null, null, null);
         logger.info("... Created Policy : " + policyBetaName);
 
-        createCondition("TestClientRolesCondition-beta", TestClientRolesConditionFactory.PROVIDER_ID, null, (ComponentRepresentation provider) -> {
+        createCondition("ClientRolesCondition-beta", ClientRolesConditionFactory.PROVIDER_ID, null, (ComponentRepresentation provider) -> {
```

Comment:
Suggest to introduce constant for `"ClientRolesCondition-beta"`

Reply:
@valb3r Thank you. I'll revise codes as you suggested.